resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/KubernetesConfSuite.scala

```diff
@@ -218,6 +218,14 @@ class KubernetesConfSuite extends SparkFunSuite {
     assert(driverConf.schedulerName === "driverScheduler")
   }
 
+  test("SPARK-37735: access appId in KubernetesConf") {
+    val sparkConf = new SparkConf(false)
+    val execConf = KubernetesTestConf.createExecutorConf(sparkConf)
+    val driverConf = KubernetesTestConf.createExecutorConf(sparkConf)
```

Comment:
Is `createExecutorConf` a typo?

Reply:
Oh, nice catch. I missed this. Thanks, @williamhyun .

To @Yikun , please fix this by using `createDriverConf` here.

Reply:
And, if possible, please put `driverConf` before `execConf`.

Reply:
Ah, thanks, will fix soon.

Reply:
done